specification/msi/ManagedIdentity.Management/models.tsp

```diff
@@ -0,0 +1,277 @@
+@doc("Paged collection of {name} items", Resource)
+model Page<Resource extends TypeSpec.Reflection.Model> {
+  @doc("The {name} items on this page", Resource)
+  @pageItems
+  value?: Resource[];
+
+  @doc("The link to the next page of items")
+  @nextLink
+  nextLink?: ResourceLocation<Resource>;
+}
+
+/**
+ * Values returned by the List operation.
+ */
+model UserAssignedIdentitiesListResult is Page<Identity>;
+
+/**
+ * The properties associated with the user assigned identity.
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "FIXME: added in TypeSpec migration, follow aka.ms/tsp/conversion-fix for details"
+@summary("User Assigned Identity properties.")
+model UserAssignedIdentityProperties {
+  /**
+   * The id of the tenant which the identity belongs to.
+   */
+  @visibility(Lifecycle.Read)
+  tenantId?: Azure.Core.uuid;
+
+  /**
+   * The id of the service principal object associated with the created identity.
+   */
+  @visibility(Lifecycle.Read)
+  principalId?: Azure.Core.uuid;
+
+  /**
+   * The id of the app associated with the identity. This is a random generated UUID by MSI.
+   */
+  @visibility(Lifecycle.Read)
+  clientId?: Azure.Core.uuid;
+
+  /**
+   * Enum to configure regional restrictions on identity assignment, as necessary.
+   */
+  @visibility(Lifecycle.Read, Lifecycle.Create, Lifecycle.Update)
+  isolationScope?: IsolationScope;
+}
+
+/**
+ * Describes an identity resource.
+ */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "FIXME: added in TypeSpec migration, follow aka.ms/tsp/conversion-fix for details"
+model IdentityUpdate extends Azure.ResourceManager.Foundations.Resource {
+  /**
+   * The geo-location where the resource lives
+   */
+  @visibility(Lifecycle.Read, Lifecycle.Create)
+  location?: string;
+
+  /**
+   * Resource tags
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-no-record" "FIXME: added in TypeSpec migration, follow aka.ms/tsp/conversion-fix for details"
+  @visibility(Lifecycle.Read, Lifecycle.Create, Lifecycle.Update)
+  tags?: Record<string>;
+
+  /**
+   * The properties associated with the identity.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-private-usage" "FIXME: added in TypeSpec migration, follow aka.ms/tsp/conversion-fix for details"
+  @Azure.ResourceManager.Private.conditionalClientFlatten
+  properties?: UserAssignedIdentityProperties;
+}
+
+/**
+ * Values returned by the List operation for federated identity credentials.
+ */
+model FederatedIdentityCredentialsListResult
+  is Page<FederatedIdentityCredential>;
+@@extension(FederatedIdentityCredentialsListResult.value,
+  "x-ms-identifiers",
+  #["id"]
+);
+
+/**
+ * The properties associated with a federated identity credential.
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "FIXME: added in TypeSpec migration, follow aka.ms/tsp/conversion-fix for details"
```

Comment:
Why we need this suppression?

Reply:
<img width="772" height="173" alt="image" src="https://github.com/user-attachments/assets/1633037f-03cf-4f8f-a20e-d64783f7e850" />